feature-libs/product-configurator/rulebased/components/attribute/types/single-selection-bundle-dropdown/configurator-attribute-single-selection-bundle-dropdown.component.ts

```diff
@@ -99,4 +99,39 @@ export class ConfiguratorAttributeSingleSelectionBundleDropdownComponent
       this.selectionChange.emit(event);
     }
   }
+
+  getSelectedValuePrice(
+    attribute: Configurator.Attribute
+  ): Configurator.PriceDetails | undefined {
+    return attribute?.values?.find((value) => value?.selected)?.valuePrice;
+  }
+
+  getSelectedValuePriceTotal(
+    attribute: Configurator.Attribute
+  ): Configurator.PriceDetails | undefined {
+    return attribute?.values?.find((value) => value?.selected)?.valuePriceTotal;
```

Comment:
quite similar to configurator-attribute-multi-selection-bundle.component.ts as said we should move this to normalizer.

Reply:
The method **getSelectedValuePriceTotal** has been removed the method **getSelectedValuePrice** has not been removed yet. The corresponding data should be first prepared by the normalizer at the attribute level

Reply:
Added a price service to avoid code duplication